arviz/stats/stats.py

```diff
@@ -719,6 +719,9 @@ def psislw(log_weights, reff=1.0):
     """
     Pareto smoothed importance sampling (PSIS).
 
+    Note that psislw will only act on the last dimension if the log_weights input is an
+    xarray.dataarray without a `__sample__` dimension or if it is a numpy array.
```

Comment:
```suggestion
    Notes
    -----
    If the ``log_weights` input is an :class:`~xarray.DataArray` with a dimension named ``__sample__`` (recommended)
    ``psislw`` will interpret this dimension as samples, and all other dimensions as dimensions of the observed data,
    looping over them to calculate the psislw of each observation. If no ``__sample__`` dimension is present or
    the input is a numpy array, the last dimension will be interpreted as ``__sample__``.
```

Reply:
Thanks, this is definitely more appropriate.